application/src/main/java/com/musicforall/model/Track.java

```diff
@@ -127,11 +129,10 @@ public boolean equals(Object obj) {
     @Override
     public String toString() {
         return "Track{" +
-                "id=" + id +
-                ", tags=" + tags +
+                "id='" + id + '\'' +
+                "tags='" + tags + '\'' +
                 ", name='" + name + '\'' +
                 ", location='" + location + '\'' +
-                ", playlists=" + playlists +
```

Comment:
Yeah, I've encountered that, too.  It causes something like infinite mutual calls, since the Playlist.toString(), in its turn, calls the Track.toString()